src/richie/plugins/large_banner/models.py

```diff
@@ -31,6 +34,21 @@ class LargeBanner(CMSPlugin):
         blank=True,
     )
     logo = FilerImageField(
-        related_name="logo", verbose_name=_("logo"), on_delete=models.PROTECT
+        related_name="logo",
+        verbose_name=_("logo"),
+        on_delete=models.PROTECT,
+        null=True,
+        blank=True,
     )
-    logo_alt_text = models.CharField(max_length=255)
+    logo_alt_text = models.CharField(max_length=255, blank=True)
+    template = models.CharField(
+        _("Template"),
+        max_length=150,
+        choices=LARGEBANNER_TEMPLATES,
+        default=LARGEBANNER_TEMPLATES[0][0],
+        help_text=_("Optional template for custom look."),
```

Comment:
this is not an optional field.
help_text=_("Choose template to render large banner plugin") ?

Reply:
It is not optional indeed, blank=False is implicit in django field, you want the explicit way ?

Reply:
I suggested to write more precise `help_text` string. The word `optional` seems wrong here